examples/1d/plot_filters_1d.py

```diff
@@ -15,7 +15,7 @@
 # First, we import the `scattering_filter_factory` method, which we will use
 # to generate the filters.
 
-from kymatio.scattering1d.filter_bank import anden_generator, scattering_filter_factory
```

Comment:
We wanted to move away from the scatnet name and all "scat" related naming, so I am not a fan of this commit.